kasa/deviceconfig.py

```diff
@@ -26,6 +26,7 @@ class EncryptType(Enum):
     Klap = "KLAP"
     Aes = "AES"
     Xor = "XOR"
+    ClearText = "CLEAR"
```

Comment:
As above I think we should keep to the existing `Aes` type and add a `bool` for `SupportsHttps` to `DeviceConfig`